docs/WindowsSubsystemForLinux.md

```diff
@@ -0,0 +1,52 @@
+# Windows Subsystem for Linux (WSL)
+- Note that all compiled Swift binaries are only executable within Bash on Windows and are Ubuntu, not Windows, executables.
```

Comment:
It might be useful here to clarify that we're talking about WSL in a *native* environment - something like "As well as being used as a cross-compile environment, Windows Subsystem for Linux can natively build and run Ubuntu executables."